CHARTER.md

```diff
@@ -108,9 +108,13 @@ process as standard proposals.
 ## Code of Conduct Enforcement
 
 Removing or disciplining an ART member due to a violation of the [Arcalot Code of
-Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is performed
-by the chairs as a pull request to the ART. Only the the chairs vote on these matters,
-with an approval by 75% of the active chairs required for the matter to pass.
+Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is initiated
+by any chair as a pull request to the ART. Only the the chairs vote on these matters,
+with an approval by unanimous consent of the active chairs and with a 7 calendar day
+voting period. In the case of a dissenting vote, unanimous consent is not achieved, and
+the matter moves to the normal proposal process as described above. An appeal may be
+made by any active ART member or by the affected party within 7 calendar days of the the
```

Comment:
"the the" again. And maybe this "appeal process" should be integrated into the review process. Instead of "only chairs vote" say that any ART member is allowed to object with reasons why the member should not be disciplined/removed, and the chairs promise to take that "appeal" feedback into consideration.

Reply:
Taking it "into consideration" doesn't give the same strength to the broader community to override the chairs. Part of the intention here is to limit the power of the chairs.